compiler-core/src/config.rs

```diff
@@ -138,6 +140,19 @@ impl PackageConfig {
             }
         })
     }
+
+    pub fn specify_hidden_modules(&self, modules: Vec<String>) -> Vec<String> {
```

Comment:
Could we get some documentation please?

I think this likely also wants to take a reference rather than ownership as it doesn't do anything other than reference and then return the data. It could return a vector of &str too